bundle/manifests/pulp-operator.clusterserviceversion.yaml

```diff
@@ -865,27 +878,31 @@ spec:
                   value: redis:latest
                 - name: RELATED_IMAGE_PULP_POSTGRES
                   value: postgres:12
-                image: quay.io/pulp/pulp-operator:v0.9.0.dev
+                image: quay.io/pulp/pulp-operator:v0.9.0
                 livenessProbe:
                   httpGet:
-                    path: /readyz
+                    path: /healthz
                     port: 6789
                   initialDelaySeconds: 15
                   periodSeconds: 20
                 name: pulp-manager
                 readinessProbe:
                   httpGet:
-                    path: /healthz
+                    path: /readyz
                     port: 6789
                   initialDelaySeconds: 5
                   periodSeconds: 10
                 resources:
                   limits:
-                    cpu: 1200m
-                    memory: 2Gi
+                    cpu: 500m
+                    memory: 768Mi
```

Comment:
:tada: 

Good call lowering the requests here.  768Mi seems oddly specific. /me expected 512Mi :man_shrugging: